topics/icrc_25_signer_interaction_standard.md

```diff
@@ -88,40 +89,18 @@ Scopes are represented in JSON-RPC 2.0 messages as JSON objects with the followi
 
 Extensions to this standard may define additional properties on scope objects.
 
-### Scopes Defined by this Standard
-
-This standard defines the wildcard (`*`) scope. It means that the relying party requests permission to invoke any method on the signer.
-
-**Example:**
-```json
-{
-    "id": 1,
-    "jsonrpc": "2.0",
-    "method": "icrc25_request_permissions",
-    "params": {
-        "scopes": [
-            {
-                "method": "*"
-            }
-        ]
-    }
-}
-```
-
-Extensions to this standard may define additional scopes.
-
 ## Extensions
 
 This standard is the signer interaction _base_ standard. As such it intentionally excludes all methods that could be handled by an extension, for example:
 
-- Getting principals: [ICRC-31](./icrc_31_get_principals.md)
+- Getting principals: [ICRC-27](./icrc_27_get_accounts.md)
 - Proving ownership of principals: [ICRC-32](./icrc_32_sign_challenge.md)
```

Comment:
Proving ownership of accounts(?)
We're not prioritizing signing a challenge right now so it's unclear what this will be. Should we remove for now?

Reply:
Yes, let's remove this for now.